requirements.txt

```
@@ -1,6 +1,7 @@
 # Base requirements for ssm
 
 argo-ams-library
+certifi<2020.4.5.2 # Used by AMS (via requests), 2020.4.5.2 dropped support for Python 2
```

```suggestion
certifi<2020.4.5.2  # Used by AMS (via requests), 2020.4.5.2 dropped support for Python 2
```
Double spaces before comments (like Python code), please.